js_modules/dagster-ui/packages/ui-core/src/asset-data/AssetLiveDataProvider.tsx

```diff
@@ -33,6 +34,8 @@ export function useAssetLiveData(assetKey: AssetKeyInput, thread: LiveDataThread
   }, [refreshBaseData, refreshStaleData]);
   const refreshing = baseDataRefreshing || staleDataRefreshing;
 
+  useBlockTraceUntilTrue('AssetLiveData', !!(baseData && staleData));
```

Comment:
We've started putting this useAssetLiveData hook INSIDE each asset node on the graph - I wonder if this will cause fan-out of trace dependencies? eg: if a page is rendering 1000 nodes, are there going to be 1000 datadog logs for that trace?

Reply:
there will be 1,000 custom timings on a single trace.